### ISSUE TITLE
Trouble wiring from storyboard to code

### ISSUE THREAD

**Issue by nickpann** [uchicago-mobi/2016-Winter-Forum#47]
I'm on "Create Table View" section, "Connect the Table Cell UI to Code".

Step 4 has us do the following: "In the editor selector bar, which appears at the top of the assistant editor, switch the assistant editor from Preview to Automatic > MealTableViewCell.swift."

However, I was unable to see MealTableCiewCell.swift in the editor selector bar. So I manually navigated to MealTableCiewCell.swift.

Then, step 11 has us control-drag from the image view on your canvas to the code display in the editor on the right.

But when I tried doing that, I couldn't make a connection. I'd control drag but the code editor wouldn't respond. I also tried writing the code and wiring to the storyboard, but that didn't work either.

So, I am trying to diagnose the cause. Why might I not be able to make a connection from the storyboard to a piece of code, and vice versa?

**Comment by danakatz** [comment]
@nickpann Were you able to connect the label in step 7 of that section? I would start by double checking the following two things:
1. Your storyboard UITableViewCell is assigned to the right class in the identity inspector, as in Design Custom Table Cells > To configure a custom cell for your table view > step 8. 
2. Your image view is nested under the table view cell in the storyboard hierarchy.

**Comment by tabinks** [comment]
@nickpann Did you figure it out?

**Comment by nickpann** [comment]
My UITableViewCell was assigned to the right class, and the image view was nested properly, so I wasn't sure what the issue is. What I ended up doing was looking at the source code of the storyboard in the Apple file and copying a segment that was different from my own source code. Then when I went to make the connection, the code editor would respond when I control dragged from the storyboard, but I got another error message. I removed and re-added MealTableViewCell.swift and it worked.
